dask_saturn/plugins.py

```diff
@@ -14,3 +20,73 @@ class SaturnSetup:
     def setup(self, worker=None):
         """This method gets called at worker setup for new and existing workers"""
         worker.scheduler.addr = resolve_address(worker.scheduler.addr)
+
+
+async def _register_files(paths=None):
```

Comment:
```suggestion
async def _register_files(paths: Optional[List[str]] = None) -> List[str]:
```

Reply:
Yeah I'll hit it with types and extend the docstrings once we've settled on a concept.